config/galaxy.ini.sample

```diff
@@ -1022,7 +1024,7 @@ use_interactive = True
 # users (email addresses).  These users will have access to the Admin section
 # of the server, and will have access to create users, groups, roles,
 # libraries, and more.  For more information, see:
-#   https://wiki.galaxyproject.org/Admin/Interface
+#   https://galaxyproject.org/admin/interface
```

Comment:
https://galaxyproject.org/admin/

P.S.: https://wiki.galaxyproject.org/Admin/Interface is missing a proper redirect.